bluesky/callbacks/zmq.py

```diff
@@ -33,7 +33,7 @@ class Publisher:
     >>> RE = RunEngine({})
     >>> publisher = Publisher('localhost:5567', RE=RE)
     """
-    def __init__(self, address, *, RE=None, zmq=None):
+    def __init__(self, address, *, RE=None, zmq=None, serializer=None):
```

Comment:
Could we default this to `pickle.dumps` directly?

Reply:
sounds great! I like it, changed.